.github/workflows/build.yml

```diff
@@ -46,6 +46,11 @@ jobs:
 
       - name: Fetch all tags and branches
         run: git fetch --prune --unshallow
+        
+      - name: Install .NET SDK
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: '6.0.x' 
```

Comment:
We aim for [deterministic builds](https://cakebuild.net/docs/writing-builds/reproducible-builds/), so we should add the exact `major.minor.patch` for all 3 SDKs we target, so there are no surprises